docs/examples/Aggregation_Data_Object_Operations.ipynb

```diff
@@ -858,47 +859,70 @@
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# we can update the georaster aggregation in HydroShare using the updated rasterio.DatasetReader object\n",
+    "# we can update the georaster aggregation in HydroShare using the updated rasterio.DatasetReader object - this operation may take a while\n",
     "gr_aggr = gr_aggr.save_data_object(resource=resource, agg_path=output_raster_dir_path, as_new_aggr=False)\n",
     "print(\"Aggregation updated ...\")"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
+   "metadata": {},
    "cell_type": "code",
-   "execution_count": null,
+   "source": [
+    "# we can also create a new georaster aggregation in HydroShare using the updated rasterio.DatasetReader object\n",
+    "\n",
+    "# If you have already updated the aggregation as described in the previous cell, then you have to first download the updated aggregation and load the\n",
+    "# rasterio.DatasetReader object from the downloaded location before you can save the updated raster to a new aggregation in HydroShare as shown below. Otherwise, you can execute the code in the next cell.\n",
+    "\n",
+    "download_to = r\"D:\\Temp\\GeoRaster_Testing\"\n",
+    "# note the unzip_to directory must exist and be empty\n",
+    "unzip_to = rf\"{download_to}\\updated_aggr_unzipped\"\n",
+    "aggr_path = resource.aggregation_download(aggregation=gr_aggr, save_path=download_to, unzip_to=unzip_to)\n",
+    "print(f\"Downloaded aggregation to:{aggr_path}\")\n",
+    "\n",
+    "# reload the updated raster as rasterio.DatasetReader\n",
+    "rasterio_ds = gr_aggr.as_data_object(agg_path=aggr_path)"
+   ],
    "outputs": [],
+   "execution_count": null
+  },
+  {
+   "metadata": {},
+   "cell_type": "code",
    "source": [
     "# we can also create a new georaster aggregation in HydroShare using the updated rasterio.DatasetReader object\n",
     "\n",
     "# we will first create a new folder in which the new aggregation will be created\n",
     "aggr_folder = \"gr_folder\"\n",
     "resource.folder_create(folder=aggr_folder)\n",
+    "# this operation may take a while\n",
     "gr_aggr = gr_aggr.save_data_object(resource=resource, agg_path=output_raster_dir_path, as_new_aggr=True,\n",
     "                                 destination_path=aggr_folder)\n",
     "print(\"New aggregation created ...\")"
    ],
-   "metadata": {
-    "collapsed": false
-   }
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
     "# retrieve the updated georaster aggregation to verify the data was updated\n",
     "\n",
     "# need to first download this updated/new aggregation\n",
+    "download_to = r\"D:\\Temp\\GeoRaster_Testing\"\n",
```

Comment:
Is this intentional? This looks like a Windows file path.

Reply:
Seems like left over from testing code. Fixed these file paths.